service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/InformantMapper.java

```diff
@@ -22,8 +22,21 @@
         if (informant.isSetAssignedEntity()) {
             Encounter.EncounterParticipantComponent component = new Encounter.EncounterParticipantComponent();
             component.setType(retrieveTypeFromITK(informant));
-            Practitioner practitioner = practitionerMapper
-                    .mapPractitioner(informant.getAssignedEntity());
+            Practitioner practitioner = practitionerMapper.mapPractitioner(informant.getAssignedEntity());
```

Comment:
Could you run reformat file please

Reply:
I did and my code formatter is set to align as you see them now. This is a code style rather than code review issue.

Reply:
We will be introducing checkstyle and dedicated IJ formatter soon. So let's leave it as it is now.